overrides/groovy/postInit/Post-Initial/Main/General/Early-Game/earlygame.groovy

```diff
@@ -104,3 +104,21 @@ if (LabsModeHelper.expert) {
 		.buildAndRegister()
 
 }
+
+// Coated Circuit Board
+mods.gregtech.assembler.recipeBuilder()
+	.inputs(ore('plateWood'))
+	.fluidInputs(fluid('glue') * 100)
+	.circuitMeta(3)
+	.outputs(metaitem('board.coated'))
+	.duration(100).EUt(VA[ULV])
+	.buildAndRegister()
+
+mods.gregtech.assembler.removeByOutput([metaitem('circuit_board.basic')], null, null, null) 
+mods.gregtech.assembler.recipeBuilder()
+	.inputs(metaitem('board.coated'))
+	.inputs(item('gregtech:meta_foil', 25) * 4)
```

Comment:
Here, use `.inputs(metaitem('board.coated'), ore('foilCopper') * 4)`

(Replacing Lines 119 and 120)

Reply:
re tested